### PR TITLE
Update enforcing-ssl.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@01binary  please review

### CODE REVIEW

**Comment by 01binary** [comment]
@Rick-Anderson any chance we could have the last section "Set up IIS Express for SSL/HTTPS" reference https://docs.microsoft.com/en-us/aspnet/core/security/https topic instead of duplicating the beginning of that topic along with the project properties screenshot?

For example:

## Set up IIS Express for SSL/HTTPS

Projects created using **Web Application** or **Web API** project templates for ASP.NET Core 2.x are automatically configured to enable SSL and launch with https url if the Individual User Accounts option was selected on Change Authentication dialog in the project wizard.

Learn how to setup SSL manually by following the steps in [Setting up HTTPS for development in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/https) topic if porting a project from an earlier version of ASP.NET Core.

**Comment by Rick-Anderson** [comment]
@01binary  sure I'll do that. See #3804

**Comment by Rick-Anderson** [comment]
@01binary  or I'll fix it in this PR

**Comment by 01binary** [comment]
My suggestion deals with content being modified in this PR (i.e. instead of inserting a Note in the beginning of the last section, I suggested to replace the last section instead). I think it makes sense to make the change as another commit in this PR if you agree with the change (perhaps a quick QA pass for wording).

I will do a quick check on security/https topic to make sure it works for Kestrel users on Mac. It would be amazing if it started off with "If you are on PC, use IIS and jump to this sub-heading and if you are on a Mac, use Kestrel and jump to this sub-heading" intro, but I can't go too far out of scope to change other topics not directly related to social logins.

File: aspnetcore/security/https.md [hunk 1]
```diff
@@ -19,8 +19,15 @@ uid: security/https
 
 You can configure your application to use HTTPS during development to simulate HTTPS in your production environment. Enabling HTTPS may be required to enable integration with various identity providers (like [Azure AD](https://azure.microsoft.com/services/active-directory) and [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c)).
 
+<a name="iisxpress"></a>
+
 On Windows if you’ve installed Visual Studio or IIS Express, the IIS Express Development Certificate will be in your LocalMachine certificate store. You can update your project properties in Visual Studio to use this certificate when running behind IIS Express.
 
+   * In Solution Explorer, right click the project and select **Properties**.
```
Comment (scottaddie) [inline aspnetcore/security/https.md]: Hyphenate "right click".

File: aspnetcore/security/https.md [hunk 2]
```diff
@@ -19,8 +19,15 @@ uid: security/https
...
+   * On the left pane, select **Debug**.
+   * Check **Enable SSL**
+   * Copy the SSL URL and paste it into the **App URL**
+
 ![Debug tab of web application properties](enforcing-ssl/_static/ssl.png)
 
 You can also configure Kestrel to listen over HTTPS by configuring an endpoint with the desired IP address, port, and certificate. The certificate can be configured inline, or in the top level `Certificates` section and then referenced by name:
```
Comment (scottaddie) [inline aspnetcore/security/https.md]: Hyphenate "top level".

**Comment by Rick-Anderson** [comment]
@01binary  take a look now.

**Comment by 01binary** [comment]
Kestrel instructions start right after the Project Properties screenshot. To minimize going back and forth between Mac/PC we could move the "If you are using macOS or Linux,..." paragraph to just after the screenshot. Thus the flow will be:

1. Intro
2. On Windows, .... [ short instructions ]
3. On Mac or Linux, ... [ longer, more manual instructions ]
4. On Max or Linux, to generate a certificate.... [ additional instructions]

rather than...

1. Intro
2. On Mac or Linux, jump to .... ---- context switch to Mac
3. On Windows, ... [short instructions] ---- context switch to Windows
4. On Mac or Linux, ... [longer instructions] --- context switch back to Mac
5. On Max or Linux, ... [additional instructions]

Just my opinion of course, can be left as-is.

**Comment by Rick-Anderson** [comment]
@01binary  is this closer?

**Comment by 01binary** [comment]
Looks good. I lost some time because Core 2.0 required OS update on my Mac so I still need to review SSL instructions for Kestrel. But if you are confident those work then I have no further comments.

**Pull request merged**